pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc {
```

Comment:
grpc-store

Reply:
.protoc file doesn't accept **grpc-store** So named it as grpc_store